Assets/HoloToolkit/UX/Examples.meta

```diff
@@ -0,0 +1,10 @@
+fileFormatVersion: 2
+guid: 57ccc1d189d25a54db4964d7458ea25b
```

Comment:
Seems like this folder got removed but this meta file still remains. Please remove